numpyro/infer_util.py

```diff
@@ -120,6 +121,23 @@ def potential_energy(model, model_args, model_kwargs, inv_transforms, params):
     return - log_joint
 
 
+def transformed_potential_energy(potential_energy, inv_transform, z):
+    """
+    Given a potential energy `p(x)`, compute potential energy of `p(z)`
```

Comment:
+1 on the choice of variable names and this docstring. Its surprising how much can keeping documentation and variable naming straight help notice bugs like this.